core/CHANGELOG.T2B1.md

```
@@ -1295,10 +1314,19 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 [#6506]: https://github.com/trezor/trezor-firmware/pull/6506
 [#6524]: https://github.com/trezor/trezor-firmware/pull/6524
 [#6551]: https://github.com/trezor/trezor-firmware/pull/6551
+[#6567]: https://github.com/trezor/trezor-firmware/pull/6567
 [#6589]: https://github.com/trezor/trezor-firmware/pull/6589
+[#6597]: https://github.com/trezor/trezor-firmware/pull/6597
+[#6601]: https://github.com/trezor/trezor-firmware/pull/6601
+[#6620]: https://github.com/trezor/trezor-firmware/pull/6620
+[#6707]: https://github.com/trezor/trezor-firmware/pull/6707
+[#6710]: https://github.com/trezor/trezor-firmware/pull/6710
+[#6780]: https://github.com/trezor/trezor-firmware/pull/6780
+[#6807]: https://github.com/trezor/trezor-firmware/pull/6807
```

markdownlint-cli2 (0.22.1)

[warning] Link and image reference definitions should be needed
Unused link or image reference definition: "`#6807`"

(MD053, link-image-reference-definitions)